package.json

```diff
@@ -27,6 +27,7 @@
     "handlebars": "^4.0.10",
```

Comment:
@SimonLab we cannot remove the "_old_" dependencies from `package.json` until we _fully_ migrate the website else the "Production" website will fail. 👍